# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/PWebrtcTCPSocket.ipdl
# Commit: 8196045d38b5
# Full Hash: 8196045d38b50e04313725215c36ba1618c50486
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Create an ipdl struct called WebrtcProxyConfig, to cut down on the arg count on lots of functions. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45562
# ==============================================================================

diff -r 6e1eba5c88ad -r 8196045d38b5 media/mtransport/ipc/PWebrtcTCPSocket.ipdl
--- a/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Wed Sep 18 14:27:42 2019 +0000
+++ b/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Tue Sep 17 18:19:04 2019 +0000
@@ -8,7 +8,8 @@
 include protocol PNecko;
 include protocol PSocketProcess;
 
-include NeckoChannelParams;
+include WebrtcProxyConfig;
+
 using mozilla::dom::TabId from "mozilla/dom/ipc/IdType.h";
 
 namespace mozilla {
@@ -24,9 +25,7 @@
                   nsCString aLocalAddress,
                   int32_t aLocalPort,
                   bool aUseTls,
-                  LoadInfoArgs? aLoadInfoArgs,
-                  nsCString? aAlpn,
-                  int? proxyPolicy);
+                  WebrtcProxyConfig? aProxyConfig);
   async Write(uint8_t[] aWriteData);
   async Close();
 